src/Features/Editor/Configure/Configure.tsx

```diff
@@ -87,79 +88,84 @@ const ConfigureContainer = React.memo<ConfigureContainerProps>(function Configur
   };
 
   return (
-    <Formik
-      enableReinitialize
-      onSubmit={handleOnSubmit}
-      initialValues={{
-        description: summaryData.description ?? "",
-        enableACCIntegration: summaryData.enableACCIntegration ?? false,
-        enablePersistentStorage: summaryData.enablePersistentStorage ?? false,
-        icon: summaryData.icon ?? "",
-        name: summaryData.name ?? "",
-        selectedTeam: teams.find((team) => team.id === params.teamId) ?? { id: "" },
-        shortDescription: summaryData?.shortDescription ?? "",
-        triggers: {
-          manual: {
-            enable: summaryData.triggers?.manual?.enable ?? true,
+    <>
+      <Helmet>
+        <title>{`Configure - ${summaryData.name}`}</title>
```

Comment:
When I open the Editor in the default workflow tab, it shows "Configure - workflowName" in the helmet instead of "Workflow - workflowName". This is fixed after switching tabs.